protocol/consistencychecks.go

```diff
@@ -275,34 +292,15 @@ func (cc *ConsistencyChecks) verifyFulfilledPromise(uname string,
 //
 // 	If the request is a key lookup, and
 // 	- the request is successful, then the directory should return a promise for the lookup binding.
-func (cc *ConsistencyChecks) verifyReturnedPromise(requestType int,
-	responseCode ErrorCode, df *DirectoryProof,
+// These above checks should be performed before calling this method.
+func (cc *ConsistencyChecks) verifyReturnedPromise(df *DirectoryProof,
 	uname string, key []byte) error {
```

Comment:
The `uname` parameter seems to be unused, and the `ap.LookupIndex` will have already been validated when `verifyReturnedPromise()` is called, right? We can drop this param.